.github/ISSUE_TEMPLATE/release.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Release
+about: When ready to cut a release
+title: Release X.Y.Z
+labels: release
+assignees: ''
+
+---
+
+- [ ] Start a new release branch:
+```bash
+$ git flow release start X.Y.Z
+```
+- [ ] Rotate `CHANGELOG.md` (following [Keep a Changelog](https://keepachangelog.com/) principles)
+- [ ] Ensure outstanding changes are committed:
+```bash
+$ git status # Ensure we have a clean staging area
+$ git add CHANGELOG.md
+$ git commit -m "X.Y.Z"
+```
+- [ ] Publish the release branch:
+```bash
+$ git flow release publish X.Y.Z
+```
+- [ ] Test the release branch on staging
+- [ ] Start a new [release pipeline job](http://civicci01.internal.azavea.com/view/oar/job/Open%20Apparel%20Registry%20Release%20Pipeline/build?delay=0sec) with the SHA of `release/X.Y.Z` that was tested on staging 
+    - `$ git rev-parse --short HEAD`
```

Comment:
It is a little ambigious what this command is for. I added a suggested change to the line above with a parenthetical. I also think it would be good to make this command render in a block like the others:

```bash
$ git rev-parse --short HEAD
```